lib/gmock-1.7.0/gtest/test/gtest-test-part_test (1).cc

```diff
@@ -0,0 +1,208 @@
+// Copyright 2008 Google Inc.
```

Comment:
I think you've confused git branches and GitHub forks. They're not the same thing. Forking creates a clone of the git repository on GitHub's server; branching creates a new place to work within a repository. Generally, the workflow for contributing an update is to create a new branch in your local git repository (which was forked from mixxxdj/mixxx originally), merge the latest changes from mixxxdj/mixxx's master branch into your new branch, then commit your changes and open a pull request.

But yeah, I can make a clean branch and submit a PR for you.

http://xkcd.com/1597/ :P